internal/operands/template-validator/reconcile.go

```diff
@@ -2,6 +2,7 @@ package template_validator
 
 import (
 	"fmt"
+
```

Comment:
This was probably changed by the IDE, but nothing else is changed in this file.
Can you remove this change?

Reply:
ok